saas/security/ir.model.access.csv

```diff
@@ -4,3 +4,4 @@ access_saas_template_operator,access_saas_template_operator,model_saas_template_
 access_saas_log,access_saas_log,model_saas_log,base.group_user,1,0,1,0
 access_saas_operator,access_saas_operator,model_saas_operator,base.group_user,1,1,1,0
 access_saas_template,access_saas_template,model_saas_template,base.group_user,1,1,1,1
+access_saas_module,access_saas_module,model_saas_module,base.group_user,1,1,1,1
```

Comment:
Add TODO to readme:

``TODO: create saas access levels (SaaS: User, Manager)``